posthog/temporal/data_imports/pipelines/postgres/postgres.py

```diff
@@ -0,0 +1,228 @@
+import dataclasses
+from typing import Any, Optional
+from collections.abc import Iterator
+import psycopg.rows
+import pyarrow as pa
+import psycopg
+from psycopg import sql
+
+from posthog.temporal.data_imports.pipelines.helpers import incremental_type_to_initial_value
+from posthog.temporal.data_imports.pipelines.pipeline.typings import SourceResponse
+from posthog.temporal.data_imports.pipelines.pipeline.utils import table_from_iterator
+from posthog.warehouse.models import IncrementalFieldType
+
+from dlt.common.normalizers.naming.snake_case import NamingConvention
+
+
+def _build_query(
+    schema: str,
+    table_name: str,
+    is_incremental: bool,
+    incremental_field: Optional[str],
+    incremental_field_type: Optional[IncrementalFieldType],
+    db_incremental_field_last_value: Optional[Any],
+) -> sql.Composed:
+    query = sql.SQL("SELECT * FROM {}.{}").format(sql.Identifier(schema), sql.Identifier(table_name))
```

Comment:
nit: You could just:
```suggestion
    query = sql.SQL("SELECT * FROM {}").format(sql.Identifier(schema, table_name))
```

Reply:
This pattern repeats a few times.

Reply:
Love it